test/Microsoft.Extensions.WebEncoders.Tests/project.json

```diff
@@ -1,13 +1,12 @@
 {
   "dependencies": {
-    "dotnet-test-xunit": "1.0.0-*",
+    "dotnet-test-xunit": "2.2.0-*",
     "Microsoft.Extensions.DependencyInjection": "1.1.0-*",
     "Microsoft.Extensions.WebEncoders": "1.1.0-*",
-    "xunit": "2.1.0"
+    "xunit": "2.2.0-*"
   },
   "testRunner": "xunit",
   "buildOptions": {
-    "allowUnsafe": true,
```

Comment:
The type is in the src package and still requires it. The test types in this code are pretty safe :)